src/frontends/onnx/tests/models/int4_const.prototxt

```diff
@@ -0,0 +1,41 @@
+ir_version: 10
```

Comment:
may be it is better to add example models with Add operation (parameter and Const)?

Reply:
I propose to create such test and disable it for now, referring CVS ticket requesting CPU support